parsons/databases/redshift/redshift.py

```diff
@@ -760,6 +760,9 @@ def upsert(self, table_obj, target_table, primary_key, vacuum=True, distinct_che
                 A temp table is dropped by default on cleanup. You can set to False for debugging.
             alter_table: boolean
                 Set to False to avoid automatic varchar column resizing to accomodate new data
+            alter_table_cascade: boolean
+                Will drop dependent objects when attempting to alter the table. If ``alter_table``
+                is not ``True``, this will be ignored.
```

Comment:
Really minor quibble but I was a little confused reading this... 

It might be easier to say "if `alter_table` is `False`,  ..."?

Reply:
Yeah, that was confusing to me too -- I copied this language from the `copy()` docs, and I can change it in both places. I think what this is trying to say is "if `alter_table` is`False` or not explicitly set to `True`", since `alter_table` is False by default.